pfs.ipmu.jp/usage.rst

```diff
@@ -0,0 +1,326 @@
+======
+
+SuMIRe/PFS PBworks wiki
+  Access `password reset <https://my.pbworks.com/?p=forgot>`_ and follow an instruction shown.
+pfs.ipmu.jp (https)
+  Send email to pfs at pfs.ipmu.jp from email address you registered.
+PFS JIRA
+  If you have an account under single sign on at pfs.ipmu.jp (https), 
+  send email to pfs at pfs.ipmu.jp. Otherwise access 
+  `JIRA registration page <https://pfspipe.ipmu.jp/jira/secure/ForgotLoginDetails.jspa>`_
+  and follow an instruction shown. 
+
+
+Details of the services
+******
+
+SuMIRe/PFS PBworks wiki
+------
+
+"SuMIRe/PFS PBworks" is the only official wiki system for PFS. 
+
+======
+Login and ToC
+======
+
+After logging in at `PBworks <http://sumire.pbworks.com/>`_, 
+you will get a project index page named 
+"Subaru Measurement of Images and Redshifts (SuMIRe)". 
+This page contains links to contents in the wiki, such as 
+
+* PFS Project Office (General links, including link to telecon indexes)
+* Documents (Link list to documents)
+* Meeting, conference, etc.  (Link list to meetings)
+* Mailing lists (List of avail lists)
+* PFS working groups (Member list)
+
+======
+Page indexes
+======
+
+For some continuous meetings, index for each agenda/memo pages are avail.
+
+* `Systems Engineering group telecon <http://sumire.pbworks.com/System-engineer-group-telecon>`_
+* `Manager group telecon <http://sumire.pbworks.com/Manager-group-telecon>`_
+
+
+======
+Files
+======
+
+Also, you can find all files uploaded into this SuMIRe/PFS PBworks from 
+`folder listing <http://sumire.pbworks.com/w/browse/#view=ViewAllFiles>`_.
+Some files are categorized into *FOLDERS*, and you can get each list by 
+clicking FOLDER name at left side. 
+
+======
+Editing manual
+======
+
+You can find `manual for editing PBworks <http://usermanual.pbworks.com/>`_. 
+
+
+Public Web Site at pfs.ipmu.jp
+------
+
+Public project information is at http://pfs.ipmu.jp , 
+such as list of meetings, list of publications, and instrument parameters. 
+
+Internal Web Site -- pfs.ipmu.jp (https)
+------
+
+Every contents at https://pfs.ipmu.jp/ are project only, and you will 
+be required to log in to view pages. 
+
+If you have any issue on this site, contact administrator 
+at pfs@pfs.ipmu.jp . 
+
+======
+Login and user account
+======
+
+Use your 'account name' (not email address) and 'password'.
+For your first time, please follow notification email to change your password 
+from an initial one (randomly created). 
+
+You can edir your account information from `LDAP account manipulator service <https://pfs.ipmu.jp/ldap-manip/>`_, 
+such as password, your real name, institution, and photo. 
+Also you can view list of all accounts from 
+
+* `List of existing accounts <https://pfs.ipmu.jp/ldap-manip/view_all.cgi>`_
+* `photo directory <https://pfs.ipmu.jp/ldap-manip/view_allphoto.cgi>`_
+
+======
+ToC
+======
+
+When accessing to https://pfs.ipmu.jp/, you will get newest list of 
+contents in this server. 
+
+LDAP account manipulator
```

Comment:
I think it is the same size as one previous - 'Login and user account'.
I don't think we can do something, since its display design is defined by github, but yeah, I understand concern of hx size for h4. For itemize or dt/dl, I suppose current one could be easier to tell than just itemize... Or we may be better to have itemize with bold heading?